bluetooth/src/androidLibMain/kotlin/Characteristic.kt

```diff
@@ -40,6 +40,8 @@ actual interface CharacteristicWrapper {
     fun intValue(formatType: Int, offset: Int): Int
 }
 
+fun CharacteristicWrapper.hasProperty(property: Int) = properties and property != 0
```

Comment:
is this correct?

Reply:
I think so, 'and' is a bitwise operator. I saw something like  properties and property > 0, but it should make a difference in this case

Reply:
It gives true also if only one of the bits set in `property` is also set in properties. But since it has a narrow case it's fine. It will be confusing no matter what.

Reply:
Doesn't look readable for me, why not to go old school:
```
properties.and(property) == property
```

Reply:
Don't see why it is more readable than the current implementation, but I don't mind changing it. 
Updated

Reply:
it's not about readability, but correctness.

`hasProperty(PROPERTY_INDICATE or PROPERTY_READ) // checks BOTH READ and INDICATE are present` gives a different result. If you're used to bitflags you'd expect it to be implement it as @avdyushin says

If you read it naively then you'd actually expect it to work the way it does now.

Reply:
renamed to `containsAnyOf` and changed signature to make it more readable